### DISCUSSION TITLE
Threats Manager Studio Support on Apple MAC

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#37] [Q&A] [answered]
Hi ,
From the download ([link](https://threatsmanager.com/downloads/))-     I understand that threats manager studio is being tested to run on linux.  Are there any plans for supporting Apple MAC ?

**Comment by simonec73** [comment, marked as answer]
Threats Manager Studio is not tested for Linux or Mac. There are parts of it that I expect to work only on Windows. 
Some tools may support Linux. I haven't prioritized the tests, but they should work. I am referring for example to the bulk conversion tool from TMT. I cannot exclude that those tools may work for Apple Mac. I even expect them to work. But I do not own a Mac. Therefore, I cannot test anything on macOS, and even less guarantee compatibility.

**Reply by rsrinivasanhome** [reply]
thanks

**Comment by simonec73** [comment]
An update on this topic: I have received confirmation from a colleague using a Mac that Threats Manager is incompatible with the latest Macs based on Mx CPUs.
